ui-src/README.md

```diff
@@ -0,0 +1,27 @@
+# Angular
```

Comment:
remove any of the auto-generated readme that isn't directly relevant to this project